### DISCUSSION TITLE
Train the model for a few specific keywords for their distinctive pronunciation!

### DISCUSSION THREAD

**Discussion by ehzawad** [coqui-ai/TTS#1509] [General Q&A]
Recently, I have trained a model for the Bengali language. And the model has been working fine, but it cannot seem to produce the correct pronunciation of a few keywords. How could retrain or reinforce the model to work it correctly on those keywords?

The complexity has been aroused due to the difference in pronunciation of a same character. Now I would train the model using the pronunciation I wanted, but as the entire model trained on a single voice (from audio-books of a particular reader), I cannot afford to train those particular words with that speaker's voice. 

Any pointers on this issue would be helpful.

**Comment by p0p4k** [comment]
Similar to #1508 , try to read about phonemes. 

A 'phonemizer' converts written text into text that is read (vocal sounds). Check [this](TTS/tts/utils/text/japanese/phonemizer.py) file, a similar function for your language can be used. For example, a quick google search produced [this](https://github.com/rahmanabir/BanglaNLP/blob/master/data_dekhabet.py).

**Reply by erogol** [reply]
Out of topic but can you reach out to me on Element or Gitter? We have a present for you :)

**Reply by p0p4k** [reply]
@erogol sure, I sent you a text on gitter.
